mysite/urls.py

```diff
@@ -15,9 +15,11 @@
 """
 from django.urls import path, include
 from django.contrib import admin
-from mercury.resources import SimulatedDataResource
 
-mercury_api = SimulatedDataResource()
+# TODO ask about the api
+from mercury.resources import TemperatureSensorResource
+
+mercury_api = TemperatureSensorResource()
```

Comment:
I can help with this part. It is a way of modeling REST resources after our data model. Before, I called it mercury_api since we just had a single entity to model.